eslint/eslint-patch/src/eslint-bulk-suppressions/cli/utils/get-eslint-cli.ts

```diff
@@ -3,27 +3,50 @@
 
 import path from 'path';
 
-const TESTED_VERSIONS: string[] = ['8.6.0', '8.7.0', '8.21.0', '8.22.0', '8.23.0', '8.23.1'];
+const TESTED_VERSIONS: Set<string> = new Set(['8.6.0', '8.7.0', '8.21.0', '8.22.0', '8.23.0', '8.23.1']);
```

Comment:
Eventually we should probably add a test using the oldest ESLint 8.6.0

Reply:
Added tests for the oldest and newest versions.